app/server/utils/ussd/token_processor.py

```diff
@@ -43,8 +43,9 @@ def get_balance(user: User):
 
     @staticmethod
     def get_limit(user: User, token: Token):
-        example_transfer = CreditTransfer(transfer_type='EXCHANGE', sender_user=user, token=token, amount=0)
+        example_transfer = CreditTransfer(transfer_type='EXCHANGE', sender_user=user, recipient_user=user, token=token, amount=0)
         limits = example_transfer.get_transfer_limits()
+        db.session.delete(example_transfer)
```

Comment:
this feels super jank... we create the CreditTransfer to get the limits. if we don't delete it though, it actually inits the CreditTransfer in the db for whatever reason, and then the next time we run this again, the transfer limit check fails because we've already had 1 transaction